### ISSUE TITLE
handle tactic argument properly

### ISSUE THREAD

**Issue by ggrov** [ggrov/tinker#78]
Normal (non-env) tactics that are arguments are currently ignored by the K combinator. This has to be fixed.

**Comment by lyhlbyl** [comment]
a proper fix to this issue is to add a filed in the atomic tactic indicating the number of arguement in the tactic.  For now, we can use a provide a dummy definition, e.g.
tactic T := T;
to skip the "K"

**Comment by ggrov** [comment]
I agree. The dummy solution will have to work for now, but does not look particularly nice...

**Comment by ggrov** [comment]
By the way, how about give env tactics a type env? E.g. mytac : env instead of ENV_mytac?
